nautobot/extras/management/__init__.py

```diff
@@ -184,70 +235,84 @@ def create_custom_statuses(
             try:
                 # May fail with an IntegrityError if a status with a different slug has a name matching this one
                 # May fail with a FieldError if the Status model no longer has a slug field
-                obj, created = Status.objects.get_or_create(slug=slug, defaults=defaults)
+                obj, created = Metadata_Model.objects.get_or_create(slug=slug, defaults=defaults)
             except (IntegrityError, FieldError) as error:
                 # OK, what if we look up by name instead?
                 defaults = choice_kwargs.copy()
                 name = defaults.pop("name")
                 # FieldError would occur when calling create_custom_statuses after status slug removal
                 # migration has been migrated
-                # e.g nautobot.extras.tests.test_management.StatusManagementTestCase.test_populate_status_choices_idempotent
+                # e.g nautobot.extras.tests.test_management.StatusManagementTestCase.test_populate_metadata_choices_idempotent
                 if isinstance(error, FieldError):
                     defaults.pop("slug")
                 try:
-                    obj, created = Status.objects.get_or_create(name=name, defaults=defaults)
+                    obj, created = Metadata_Model.objects.get_or_create(name=name, defaults=defaults)
                 except IntegrityError as err:
                     raise SystemExit(
-                        f"Unexpected error while running data migration to populate status for {model_path}: {err}"
+                        f"Unexpected error while running data migration to populate {metadata_model} for {model_path}: {err}"
                     ) from err
 
             # Make sure the content-type is associated.
             if content_type not in obj.content_types.all():
                 obj.content_types.add(content_type)
 
             if created and verbosity >= 2:
-                print(f"      Adding and linking status {obj.name}", flush=True)
+                print(f"      Adding and linking {metadata_model} {obj.name}", flush=True)
                 added_total += 1
             elif not created and verbosity >= 2:
-                print(f"      Linking to existing status {obj.name}", flush=True)
+                print(f"      Linking to existing {metadata_model} {obj.name}", flush=True)
                 linked_total += 1
 
     if verbosity >= 2:
-        print(f"    Added {added_total}, linked {linked_total} status records")
+        print(f"    Added {added_total}, linked {linked_total} {metadata_model} records")
 
 
-def clear_status_choices(
+def clear_metadata_choices(
```

Comment:
I think we should do the same as `_create_custom_role_or_status_instances`:
Rename this to `_clear_custom_role_or_status_instances` and make wrappers for `clear_role_choices` and `clear_status_choices`